client/aliases_test.go

```diff
@@ -134,6 +134,32 @@ func (cs *clientSuite) TestClientRemoveManualAlias(c *check.C) {
 	})
 }
 
+func (cs *clientSuite) TestClientPreferCallsEndpoint(c *check.C) {
+	cs.cli.Prefer("some-snap")
+	c.Check(cs.req.Method, check.Equals, "POST")
+	c.Check(cs.req.URL.Path, check.Equals, "/v2/aliases")
+}
+
+func (cs *clientSuite) TestClientPrefer(c *check.C) {
+	cs.rsp = `{
+		"type": "async",
+                "status-code": 202,
```

Comment:
it seems an artefact of the code and similar code mixing tabs and spaces in string constants, not sure I want to cleanup that here. fwiw it looks correctly aligned locally